app/scripts/lib/fxa-client.js

```diff
@@ -316,19 +328,25 @@ function (FxaClient, $, p, Session, AuthErrors) {
         });
     },
 
-    sendAccountUnlockEmail: function (email, relier) {
+    sendAccountUnlockEmail: function (originalEmail, relier, options) {
       var self = this;
+      var email = trim(originalEmail);
```

Comment:
Nice catch; out of curiosity, what does this guard against?  Should we be doing it in one canonical place upstream of this code, rather tham trimming every instance of email address that's passed in to this client?